MetaMask/metamask-extension
# fix(language): typo thai ui text

## **Description**
I found thai word is incorrect
สำเหร็จ -> สำเร็จ
แอดแดรส -> ที่อยู่
เเริ่ม -> เริ่ม
แก็ส -> แก๊ส

## **Related issues**
#27839 fix wrong typo for Thai language
re-create from PR #19251 fix: typo thai ui text

## **Manual testing steps**

1. Add new account
2. นำเข้าบัญชี
3. เลือกประเภทไฟล์ -> json
then you will see wrong typo like screenshort

## **Screenshots/Recordings**
![image](https://github.com/user-attachments/assets/0f1e061c-b7c3-4f29-ab9e-0bf9a4699793)

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
